src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -45,6 +45,12 @@ object WebReportRouteConfig : RouteConfig
                     .json()
                     .secure(runReports)
                     .transform(),
+            WebEndpoint(
+                    "/reports/:name/",
```

Comment:
We've gone from `/reports/:name/parameters/:commit/` to `/reports/:name/`. I think `/reports/:name/parameters/` would be much better here, especially as this new endpoint doesn't return a report, and we already have `/report/:name/:version/`.